shell/platform/android/io/flutter/app/FlutterPluginRegistry.java

```diff
@@ -125,7 +127,8 @@ public PlatformViewRegistry platformViewRegistry() {
 
         @Override
         public FlutterView view() {
-            return mFlutterView;
+            assert mViewRegistry instanceof FlutterView;
```

Comment:
This isn't gonna fly for people using the new TextureView version - they need plugins, too...

Reply:
Are you saying that plugins will not work at all even if the plugin implementation is not calling `view()` method?

Reply:
I checked and only two (internal) plugins use this because they need to take a screenshot. That's not usable for add2app scenarios anyway where the view can be part of the screen. None of the plugins we bring in as third-party access `view()`.